PSWebServiceLibrary.php

```diff
@@ -270,32 +270,57 @@ public function add($options)
 	 * @param array $options Array representing resource to get.
 	 * @return SimpleXMLElement status_code, response
 	 */
-	public function get($options)
-	{
-		if (isset($options['url']))
+	public function get($options) {
+		$options['output_format'] = 'XML';
+		return self::parseXML(self::getRawResponse($options));
+	}
+
+	/**
+	 * get response as JSON
+	 * @param array $options
+	 * @return string
+	 */
+	public function getJson($options) {
+		$options['output_format'] = 'JSON';
+		return self::getRawResponse($options);
+	}
+
+	/**
+	 * @param array $options
+	 * @return string
+	 * @throws PrestaShopWebserviceException
+	 */
+	private function getRawResponse() {
```

Comment:
Why does the definition of `getRawResponse()` take no arguments, but the calls to this function pass `$options` ?